src/ansys/dpf/core/fields_container.py

```diff
@@ -517,11 +517,6 @@ def animate(self, save_as=None, deform_by=None, scale_factor=1.0, **kwargs):
         if frequencies is None:
             raise ValueError("The fields_container has no time_frequencies.")
 
-        loop_over = self.get_time_scoping()
-        frequencies = self.time_freq_support.time_frequencies
-        if frequencies is None:
-            raise ValueError("The fields_container has no time_frequencies.")
-
```

Comment:
I suppressed these lines because these lines were written [twice in a row](https://github.com/pyansys/pydpf-core/pull/822/files#diff-514e3347e0da0e41d5f719a5c7942de9cc250c49ba0b1defb2408503aa27a086:~:text=outputs.field-,loop_over%20%3D%20self.get_time_scoping(),raise%20ValueError(%22The%20fields_container%20has%20no%20time_frequencies.%22),-loop_over%20%3D%20self). 
Let me know if you want me to revert the change.